### ISSUE TITLE
[Issue] Orbs Issue

### ISSUE THREAD

**Issue by Pandas0** [GC-spigot/AdvancedEnchantments#1282]
**Describe the bug**
When you modify the max slots in your config the slots placeholder for the orbs seem to not get updated.

I set my slots to 5 and gave my player a 7 lore orb and the item says -2 slots when it should say +2 slots.

**To Reproduce**
Lower your max slots in your config to 5 and give yourself a 7 lore orb and look at the description

**Screenshots / Video**
https://imgur.com/fQc4TAO

**Server Information (please complete the following information):**
 - Minecraft Version: Paper 1.16.1
- Plugin Version: 7.10.2

**Comment by GC-spigot** [comment]
Show me your slots configuration

**Comment by Pandas0** [comment]
[https://imgur.com/dIw0yvM](url)

**Comment by Pandas0** [comment]
https://imgur.com/dIw0yvM

**Comment by GC-spigot** [comment]
Yeah so it works from 'max-increase-of-slots', you give a 7 slot orb and it's -2 because it's 2 less than max. If you give 10 slots orb it will give additional 1 slot

**Comment by Pandas0** [comment]
That makes no sense. Why would it run off of the max enchantment number that I want to be set by providing the orbs instead of the 'max' default slots I have set as that is what it should be increasing.

Based on what you said in the default config with 'max: 9' and 'max-increase-of-slots: 13' if I gave myself a 10 slot orb It should say -3 but instead it says +1 because it is working off of the 'max' not the 'max-increase-of-slots'. It should not give me a +1 orb until I put a 14 slot input. It seems that the plugin isn't recognizing the 'max' slots being changed and still treating it like its set to 9 when it isn't. I changed my 'max' to 5 and my 'max-increase-of-slots' to 7 and gave myself a 8 slots orb, based on what you said the orb should then say +1, but it still says -1 which is exactly what it does if I gave myself a 8 lore orb in the default config as the 'max' is set to 9 as anything lower would be a negative number, but in my case of having it set to 5 it should say +3 slots as 5 + 3 = 8.
